dev/wlp-jakartaee-transform/rules/jakarta-bundles.properties

```diff
@@ -1 +1,11 @@
-*=*.jakarta,2.0,+" Jakarta",+"; Jarkata Enabled"
\ No newline at end of file
+*=*.jakarta,2.0,+" Jakarta",+"; Jarkata Enabled"
+
+com.ibm.ws.security.authentication.tai=io.openliberty.security.authentication.tai.internal,2.0,"Security TAI Authentication","Security TAI Authentication; Jarkata Enabled"
+com.ibm.ws.security.jaspic.1.1=io.openliberty.security.jaspic.2.0.internal,2.0,"Security JASPIC 2.0 Internal","Security JASPIC 2.0 Internal; Jakarta Enabled"
+com.ibm.ws.security.javaeesec.1.0=io.openliberty.security.jakartasec.2.0.internal,2.0,"Jakarta Security 2.0 Internal","Jakarta Security 2.0 Internal"
+com.ibm.ws.security.javaeesec.cdi=io.openliberty.security.jakartasec.2.0.cdi.internal,2.0,"Jakarta Security 2.0 CDI","Jakarta Security 2.0 CDI"
+com.ibm.websphere.appserver.spi.jaspic=io.openliberty.jaspic.2.0.spi,2.0,"Jakarta Authentication Service Provider Interface for Containers 2.0 SPI ","Jakarta Authentication Service Provider Interface for Containers 2.0 SPI"
+com.ibm.ws.security.jaas.common=io.openliberty.security.jaas.common.internal,2.0,"Security JAAS Implementation","Security JAAS Implementation"
+com.ibm.ws.security.authentication.builtin=io.openliberty.security.authentication.builtin.internal,2.0,"Security Built-in Authentication Implementation","Security Built-in Authentication Implementation; Jakarta Enabled"
+com.ibm.ws.webcontainer.security=io.openliberty.webcontainer.security.internal,2.0,"Jakarta Webcontainer Security Collaborator internal","Jakarta Webcontainer Security Collaborator internal"
+com.ibm.ws.dynacache=io.openliberty.dynacache.internal,2.0,"Jakarta Dynacache","Jakarta Dynacache Core cache capability in Liberty"
```

Comment:
Dynacache is not a Jakarta function so its description should not include the word Jakarta in it.

Reply:
Yeah, i was trying to describe this as being jakarta compliant / capable.

Reply:
I changed the dynacache bundle to have "; Jakarta Enabled" at the end of the description, the same as the transformation tool does by default.